pdns/lua-record.cc

```diff
@@ -1095,19 +1096,14 @@ static void setupLuaRecords(LuaContext& lua) // NOLINT(readability-function-cogn
    *
    * @example ifportup(443, { '1.2.3.4', '5.4.3.2' })"
    */
-  lua.writeFunction("ifportup", [](int port, const boost::variant<iplist_t, ipunitlist_t>& ips, const boost::optional<std::unordered_map<string,string>> options) {
-      if (port < 0) {
-        port = 0;
-      }
-      if (port > std::numeric_limits<uint16_t>::max()) {
-        port = std::numeric_limits<uint16_t>::max();
-      }
+  lua.writeFunction("ifportup", [](int port, const boost::variant<iplist_t, ipunitlist_t>& ips, boost::optional<std::unordered_map<string, string>> options) {
```

Comment:
Still missing the `const` qualifier to `options` as it used to be.

Reply:
This doesn't matter. This `const` removal is not related to the intent of this PR, and if you restore the line to its current contents, clang-tidy should not complain about it.

In other words: that change is out of scope of this particular PR, in my opinion.

Reply:
I completely disagree. Reverted anyway.

Reply:
Let's agree to disagree.

You still have a whitespace change on that line (`<string, string>` instead of `<string,string>`) which causes `clang-tidy` to fire.